### PR TITLE
New section in changelog for calling and meeting APIs.

### PR DESCRIPTION
**Pull request by billbliss**
Also a few other tweaks to calling & meeting content. 
Bug fixes in TOC.

### CODE REVIEW

File: api-reference/beta/api/call_changescreensharingrole.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 > **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
 
-Start and stop sharing screen in the call
+Start and stop sharing screen in the call. This API is used to allow applications to share content with the participants of a call or meeting.
```
Comment (satyakonmsft) [inline api-reference/beta/api/call_changescreensharingrole.md]: "This API is used to allow applications to share content with the participants of a call or meeting." - can you make it explicit that only screen sharing content is involved here? We dont allow any other content like file sharing

File: concepts/changelog.md [hunk 2]
```diff
@@ -6,6 +6,19 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
...
+| Addition        | Beta        | Added the [call](../api-reference/beta/resources/call.md) resource for managing audio/video calls (initially, in Microsoft Teams), including APIs for [creating calls](../api-reference/beta/api/application_post_calls), [getting a list of active calls](../api-reference/beta/api/call_get), [deleting (hanging up) calls](../api-reference/beta/api/call_delete), [answering calls](../api-reference/beta/api/call_answer), [rejecting calls](../api-reference/beta/api/call_reject), [redirecting calls](../api-reference/beta/api/call_redirect), and [transferring calls](../api-reference/beta/api/call_transfer). We've also added APIs to support [IVR scenarios](../api-reference/beta/resources/calls-api-ivr-overview): [playing a prompt](../api-reference/beta/api/call_playprompt), [recording a call](../api-reference/beta/api/call_record), [cancelling media processing](../api-reference/beta/api/call_cancelmediaprocessing), and [subscribing to touch tone notifications](../api-reference/beta/api/call_subscribetotone). |
```
Comment (satyakonmsft) [inline concepts/changelog.md]: "getting a list of active calls" - not sure if we have the API. Let us double check this. We surely have one for getting info about an active call.

File: api-reference/beta/api/call_changescreensharingrole.md (hunk 1 above)
Comment (billbliss) [reply api-reference/beta/api/call_changescreensharingrole.md]: Changed.

File: concepts/changelog.md (hunk 2 above)
Comment (billbliss) [reply concepts/changelog.md]: Fixed. Made a few other wording changes too.

**Pull request merged**